Controller/PostController.php

```diff
@@ -165,8 +167,6 @@ public function archiveYearlyAction($year, Request $request = null)
      */
     public function viewAction($permalink, Request $request = null)
     {
-        $request = $this->resolveRequest($request);
```

Comment:
It was a mistake in my previous PR. The request is never used in the viewAction.